tests/features/steps/auth_vars.py

```diff
@@ -32,7 +32,7 @@
 		i] + " for 1 month"
 for i in range(6, 8):
 	acl_set_policy += "; consumer@iisc.ac.in can access example.com/test-category/" + resource_ids[
-		i] + " for 1 month if scope = read"
+		i] + " for 1 second if scope = read"
```

Comment:
How is this being used? Is it to check for expired tokens?

Reply:
This is to check for expired tokens to introspect the invalid ids